scrapy/dupefilters.py

```diff
@@ -61,8 +60,9 @@ def close(self, reason):
 
     def log(self, request, spider):
         if self.debug:
-            msg = "Filtered duplicate request: %(request)s"
-            self.logger.debug(msg, {'request': request}, extra={'spider': spider})
+            msg = "Filtered duplicate request: %(request)s (referer: %(referer)s)"
+            args = {'request': request, 'referer': referer_str(request) }
```

Comment:
Hey! Sorry for delay. No objections to me, a nice feature. I was just wondering if Referer should be added to the next message as well, when debug is False.

Reply:
Fair point.
I just added to keep the same behavior, but it makes sense to log referer only in debug mode.